sources/@roots/bud/package.json

```diff
@@ -92,8 +92,10 @@
     "html-webpack-plugin": "^5.5.0",
     "lodash": "^4.17.21",
     "mini-css-extract-plugin": "^2.4.5",
+    "ts-node": "10.4.0",
     "tslib": "^2.3.1",
     "type-fest": "^2.5.4",
+    "typescript": "4.5.5",
```

Comment:
this is the big change.

i want to spend 65mb and include `typescript` and `tsnode` as assumed dependencies of `@roots/bud`. This really smoothes out the experience with `@roots/bud-typescript`. That package is important to me.

It also lets anyone with a basic `tsconfig.json` write their config as `bud.config.ts` (without `@roots/bud-typescript`, that just becomes a feature of `@roots/bud` directly). 

TS config is a MUCH better experience. The IDE gives you so much more information.